cognite/client/_api/model_hosting/schedules.py

```diff
@@ -17,8 +17,8 @@ def create_schedule(
         """Create a new schedule on a given model.
 
         Args:
-            model_id (int): Id of model to create schedule on
-            name (str): Name of schedule
+            model_name (str): Name of model to create schedule on
+            schedule_name (str): Name of schedule
```

Comment:
If we decide to leave the others as `name`, this one should be `name` instead of `schedule_name`.

Reply:
I agree.

Reply:
Also it should maybe say "the model" and "the schedule", not sure.

Reply:
Yes, agree.